bakery-website/src/components/AboutUs/index.js

```diff
@@ -12,41 +12,41 @@ import {
   ImgWrap,
   Img,
 } from "./AboutusElements";
+import logo from "../../images/undraw_Birthday_cake_2wxy.svg";
 
 const AboutUs = ({
   lightBg,
-	id,
-	imgStart,
+  id,
+  imgStart,
   topLine,
   lightText,
   headline,
   darkText,
   description,
-  img,
   alt,
 }) => {
-    return (
-      <>
-        <AboutusContainer lightBg={lightBg} id={id}>
-          <AboutusWrapper>
-            <AboutusRow imgStart={imgStart}>
-              <Column1>
-                <TextWrapper>
-                  <TopLine>{topLine}</TopLine>
-                  <Heading lightText={lightText}>{headline}</Heading>
-                  <Subtitle darkText={darkText}>{description}</Subtitle>
-                </TextWrapper>
-              </Column1>
-              <Column2>
-                <ImgWrap>
-                  <Img src={img} alt={alt} />
-                </ImgWrap>
-              </Column2>
-            </AboutusRow>
-          </AboutusWrapper>
-        </AboutusContainer>
-      </>
-    );
-  };
+  return (
+    <>
```

Comment:
You only need to wrap your elements in empty elements (`<></>` or React.Fragments) when there is no parent. Clearly, in this case, `AboutusContainer ` is a parent and you don't need empty elements. Let me know if you are confused :)

Reply:
Yeah that's true. Don't know why I put that there. Thank you!